app/src/main/kotlin/com/google/samples/apps/nowinandroid/ui/interests2pane/InterestsListDetailScreen.kt

```diff
@@ -132,3 +153,59 @@ private fun <T> ThreePaneScaffoldNavigator<T>.isListPaneVisible(): Boolean =
 @OptIn(ExperimentalMaterial3AdaptiveApi::class)
 private fun <T> ThreePaneScaffoldNavigator<T>.isDetailPaneVisible(): Boolean =
     scaffoldValue[ListDetailPaneScaffoldRole.Detail] == PaneAdaptedValue.Expanded
+
+/**
+ * This is a direct clone of [calculateStandardPaneScaffoldDirective] with the only change of
```

Comment:
It's unfortunate that there isn't a simpler way to configure a scaffold directive.

Reply:
This is no longer needed with the update to alpha10